triple_product_vs_T/triple_product_vs_T.py

```diff
@@ -497,7 +498,7 @@ def main():
             # make annotation text (is done once for every case...)
             plt.annotate( names[ii],
                           xy=( T_vals[ii], nTtau_vals[ii]), 
-                          xytext=(2, 0.7e20),
+                          xytext=(2.2, 0.7e20),
```

Comment:
Shouldn't that rather be something like  xytext=(2.2, 0.7e19)  , otherwise the label is far off the corresponding symbol.

(Sorry for this annoying way of making the labels....)

Reply:
ho yeah, of course !... pff, tired.